### ISSUE TITLE
exit status 77 should be recognized in make check, not same as error=1 

### ISSUE THREAD

**Issue by maynardGK** [gnudatalanguage/gdl#795]
We have an example of a test procedure exiting because of lack of software.
This should not cause "make check" to fail.  (test_simplex.pro):
````
   if ~GLPK_EXISTS() then begin
      MESSAGE, /continue, 'GDL was compiled without GLPK support'
      EXIT, status=77
   endif

````

**Comment by slayoo** [comment]
This was meant to be fixed in:
https://github.com/gnudatalanguage/gdl/commit/31b2e5c3b78c5b7b53b1ab9e3f16f69f8dc54cbe

Following introduction of this feature to CMake:
https://public.kitware.com/Bug/view.php?id=8466

But apparently the launchtest.c breakes it as it "converts" the 77 exit codes into 0 here:
https://github.com/gnudatalanguage/gdl/blob/master/testsuite/CMakeLists.txt#L77

Let's see if this helps:
https://github.com/gnudatalanguage/gdl/pull/796

**Comment by slayoo** [comment]
Fixed by #796
